site-service/src/main/resources/bootstrap.yml

```diff
@@ -0,0 +1,22 @@
+spring:
+  application:
+    name: site-service
+  cloud:
+    config:
+      # If cannot connect to config server, don't start the application
+      fail-fast: true
+      discovery:
+        enabled: true
+        service-id: CONFIG-SERVER
+      retry:
+        initialInterval: 2000
+        multiplier: 1.5
+        maxInterval: 60000
+        maxAttempts: 100
+      label: main
+
+eureka:
+  client:
+    fetch-registry: true
+    serviceUrl:
+      defaultZone: http://${EUREKA_HOSTNAME:localhost}:8761/eureka/
```

Comment:
maybe the port also need to be an env var?

Reply:
same for the rest of the services

Reply:
Done